src/topology/bounded_continuous_function.lean

```diff
@@ -429,5 +429,109 @@ def of_normed_group_discrete {α : Type u} {β : Type v}
   (f : α  → β) (C : ℝ) (H : ∀x, norm (f x) ≤ C) : α →ᵇ β :=
 of_normed_group f C H continuous_of_discrete_topology
 
+/- Formula for norm. -/
+lemma norm_eq ( f : α →ᵇ β) :
+  ∥f∥ = Inf {C : ℝ | 0 ≤ C ∧ ∀ (x : α), ∥f x∥ ≤ C} :=
+begin
+  rw ← sub_zero f,
+  rw ← dist_eq_norm,
+  rw bounded_continuous_function.dist_eq,
+  simp,
+end
+
+
 end normed_group
+
+
+section normed_space
+/- In this section, if β is a normed space, then we show that the space of bounded
```

Comment:
```suggestion
/-! In this section, if `β` is a normed space, then we show that the space of bounded
```
By adding the `!`, this will become part of the auto-generated documentation.